internal/querynodev2/delegator/delegator_data.go

```diff
@@ -128,14 +128,6 @@ func (sd *shardDelegator) ProcessInsert(insertRecords map[int64]*InsertData) {
 			// panic here, insert failure
 			panic(err)
 		}
-		metrics.QueryNodeNumEntities.WithLabelValues(
```

Comment:
move it into segment `Insert` interface to keep it consistent with `segment.insertCount`